OracleGoldenGate/21c/README.md

```diff
@@ -143,13 +158,15 @@ Oracle GoldenGate Administration Client for Oracle
 Version 21.3.0.0.0 ...
 ```
 
-### Running scripts before setup and on startup
-The container images can be configured to run scripts before setup and on startup. Currently, `.sh` extensions are supported. For setup scripts just mount the volume `/u01/ogg/scripts/setup` or extend the image to include scripts in this directory. For startup scripts just mount the volume `/u01/ogg/scripts/startup` or extend the image to include scripts in this directory. Both of those locations are static and the content is controlled by the volume mount.
+### Running Scripts Before Setup and on Startup
+
+The container images can be configured to run scripts before setup and on startup. Currently, `.sh` extensions are supported. For setup scripts just mount the volume `/u01/ogg/scripts/setup` or extend the image to include scripts in this directory. For startup scripts just mount the volume `/u01/ogg/scripts/startup` or extend the image to include scripts in this directory. Both of those locations
```

Comment:
I would rephrase this as, "Containers created from this image are configured to automatically run scripts found at specific locations at particular points during initial container creation and on subsequent startups."

Then you'd have two blocks: "To run a script before the initial setup of Oracle GoldenGate ......" and  "To run a script each time the container starts but before Oracle GoldenGate is started...."

Reply:
I agree the language here is not the best, but the changes in this PR are limited to:

1. Fix capitalization on the section title
2. Shorten lines to make super-linter happy.